targets/ramips-rt305x

```diff
@@ -1,4 +1,14 @@
 config '# CONFIG_KERNEL_KALLSYMS is not set'
+config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
+
+no_opkg()
+
+packages {
+	'-kmod-usb-core',
+	'-kmod-usb2',
```

Comment:
Are you sure rt305x has kmod-usb2?

Reply:
@mweinelt Many devices add kmod-usb2 in their DEVICE_PACKAGES.

Reply:
@mweinelt do you expect an answer from my side, too?
do you want me to "add" kmod-usb-dwc2 ?

Reply:
@mweinelt you were right, not kmod-usb2 has to be removed but kmod-usb-dwc2. i did this now and also changed the PR to only affect the 4mb devices.